full.async/src/full/async.clj

```diff
@@ -62,6 +68,26 @@
              (recur (dec retries#)))
            res#)))))
 
+(defmacro go-try> [err-chan & body]
```

Comment:
I'm wondering if this could be combined into go-try? The macro could check if first argument is a channel, and use that as error channel, otherwise work as before?

Btw seems that we haven't documented it anywhere but the way we use > suffix on functions is to indicate that function returns a channel with single result. For functions that return a channel with multiple results, we use >> suffix.

A kind of exception to this is go-try/go-retry but it's a macro and 'go' prefix already makes it clear that it returns a channel. What was your thinking with > suffix?